Makefile

```diff
@@ -58,6 +58,10 @@ db:             ## Reset the database and load fixtures
 db: vendor
 	$(RUN) php -r "for(;;){if(@fsockopen('db',3306)){break;}}" # Wait for MySQL
 	$(RUN) $(CONSOLE) doctrine:database:drop --force --if-exists
+	$(RUN) $(CONSOLE) doctrine:database:create -if-not-exists --env=test_sqlite || true
```

Comment:
@Remg asked to remove these changes as we should use `make tfp` to load db for tests.